README.md

```diff
@@ -31,15 +31,17 @@ Detailed dependency installation instructions are available on the
 
 ## Running the code
 
-There are two main options for running the code:
-1. Training a model (including starting from a previously trained model using
-configuration options)
+There are three main options for running the code:
+1. Running with an existing model
+2. Training a model
     ```
-    python run_config_file.py
+    import gcnn
+    gcnn.run_normal_config_file()
     ```
-2. Hyperparameter optimization
+3. Hyperparameter optimization
```

Comment:
Since at the moment we are not considering the option to run hyper parameter optimization, maybe we should not mention it for now.

Reply:
Right, I forgot it was removed :)